src/OpenApi/perf/Microbenchmarks/TransformersBenchmark.cs

```diff
@@ -102,7 +101,7 @@ public void SchemaTransformer_Setup()
         {
             _options.AddSchemaTransformer((schema, context, token) =>
             {
-                schema.Extensions ??= [];
+                schema.Extensions ??= new Dictionary<string, IOpenApiExtension>();
```

Comment:
Are all these changes needed or did AI do this? It seems like a bunch of unnecessary changes in this PR...

Reply:
These changes are needed. The underlying API changed from being typed as `Dictionary` to `IDictionary` so we have to specify the concrete type explicitly and can't rely on collection expressions anymore.